drivers/atwinc15x0/atwinc15x0_netdev.c

```diff
@@ -144,6 +308,137 @@ typedef union {
 
 static bool _rssi_info_ready = false;
 
+static void _atwinc15x0_handle_M2M_WIFI_RESP_SCAN_DONE(tstrM2mScanDone* scan_done)
```

Comment:
Please use lower case function names here an below. Function names should be a bit shorter not to hit the 100 characters limit, for examples:
```suggestion
static void _atwinc15x0_handle_scan_done(tstrM2mScanDone* scan_done)
```